docs/builders/clone.mdx

```diff
@@ -236,6 +236,51 @@ or responding to pattern `/dev/.+`. Example:
   - `ssd` (bool) - Drive will be presented to the guest as solid-state drive
     rather than a rotational disk.
 
+- `pci_devices` (array of objects) - Passthrough a host PCI device into the VM. For example, a graphics card or a network adapter. Devices that are mapped into a guest VM are no longer available on the host. A minimal configuration only requires either the `host` or the `mapping` key to be specifed. Note: VMs with passed-through devices cannot be migrated.
+
+  Example configuration (HCL):
+
+  ```hcl
+  pci_devices {
+    host          = "0000:0d:00.1"
+    pcie          = false
+    device_id     = "1003"
+    legacy_igd    = false
+    mapping       = "someNic"
+    mdev          = "some-model"
+    rombar        = true
+    romfile       = "vbios.bin"
+    sub_device_id = ""
+    sub_vendor_id = ""
+    vendor_id     = "15B3"
+    x_vga         = false
+  }
+  ```
+
+  - `host` (string) - The PCI ID of a host’s PCI device or a PCI virtual function. You can us the `lspci` command to list existing PCI devices. Either this or the `mapping` key must be set.
```

Comment:
FYI: those options are compiled from the Go code documentation, into the partial configuration docs you've included in this PR. Those partials can be included in the documentation here, and in ISO.mdx, so you won't have to copy-paste those (which can easily diverge in the future when we modify them later).

If you want an example of this, @mabeett pushed a PR recently and did something similar in their code on PR #195: https://github.com/hashicorp/packer-plugin-proxmox/pull/195/files#diff-d395f7f43fac39fb86461a2b4239b646077ca6c2989e2fa3b0c6f174038b29ebR359